iceoryx_posh/include/iceoryx_posh/gateway/toml_gateway_config_parser.hpp

```diff
@@ -35,11 +34,16 @@ enum TomlGatewayConfigParseError
     FILE_NOT_FOUND,
     INCOMPLETE_CONFIGURATION,
     INCOMPLETE_SERVICE_DESCRIPTION,
-    INVALID_SERVICE_DESCRIPTION
+    INVALID_SERVICE_DESCRIPTION,
+    EXCEPTION_IN_PARSER
 };
 
-constexpr char TomlGatewayConfigParseErrorString[][64] = {
-    "FILE_NOT_FOUND", "INCOMPLETE_CONFIGURATION", "INCOMPLETE_SERVICE_DESCRIPTION", "INVALID_SERVICE_DESCRIPTION"};
+constexpr char TOML_GATEWAY_CONFIG_FILE_PARSE_ERROR_STRINGS[][64] = {"INVALID_STATE",
```

Comment:
this could be a `constexpr const char* TOML...STRINGS[] = {}`

Reply:
Probably a general clean-up task as there're a lot of `constexpr char` in the code base!